backend/src/services/interface.ts

```diff
@@ -0,0 +1,53 @@
+export const NETWORK_CHAINID_TO_COINID: Record<number, string> = {
+  1: 'ethereum',
+  137: 'matic-network',
+  56: 'binancecoin',
+  100: 'xdai',
+  43114: 'avalanche-2',
```

Comment:
Why is this called avalanche-2? Is this coingecko specific?

Reply:
Yes